### PR TITLE
Use smtpapi for recipient list

### PR DESCRIPTION
**Pull request by richardbrockie**
I'm proposing that multiple recipients are hidden from each other by using the smtpapi.add_tos() method.

### CODE REVIEW

**Comment by skru** [comment]
This would be really handy

File: sgbackend/mail.py [hunk 1]
```diff
@@ -65,7 +65,13 @@ def send_messages(self, emails):
 
     def _build_sg_mail(self, email):
         mail = sendgrid.Mail()
-        mail.add_to(email.to)
+        
+        # use smtpapi if recipient list provided, otherwise use standard to field
+        if len(email.to) > 1:
```
Comment (evgenyfadeev) [inline sgbackend/mail.py]: Looks like any string with more than 1 char will do here and the "tos" will be added letter by letter.
Better check for isinstance?

**Comment by richardbrockie** [comment]
Modified to explicitly test for list.

**Comment by richardbrockie** [comment]
You are not wrong - transcription error. Confirmed that the method is .set_tos()

**Comment by elbuo8** [comment]
This could be misleading to the users IMO.

**Pull request closed without merging**